### PR TITLE
readme update

### PR DESCRIPTION
**Pull request by ceeram**
i like this better

demo

### CODE REVIEW

**Comment by renan** [comment]
Denied!

**Comment by ceeram** [comment]
why?

File: readme.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ## Setup (ubuntu)
 
 ```sh
-aptitude install git-core php5-cli php5-curl
+apt-get install git-core php5-cli php5-curl
```
Comment (renan) [inline readme.md]: aptitude also cleans old/uneeded packages

**Comment by JustInVTime** [comment]
Because we can!

**Pull request closed without merging**